crates/aranya-cli/testing/build-teams.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+# Create directories for all daemons
+for i in {1..5}; do
+    mkdir -p /tmp/aranya$i/{run,state,cache,logs,config}
+done
+
+echo "=== Creating teams on each daemon ==="
+
+# Array to store team info for each daemon
+declare -a TEAM_IDS
+declare -a DEVICE_IDS
+declare -a IDENTITY_KEYS
+declare -a SIGNING_KEYS
+declare -a ENCODING_KEYS
+
+# Create team on each daemon
+for i in {1..5}; do
+    echo "Creating team on daemon $i..."
+    
+    # Create team
+    TEAM_OUTPUT=$(aranya --uds-path /tmp/aranya$i/run/uds.sock -v create-team)
+    TEAM_ID=$(echo "$TEAM_OUTPUT" | grep "Team ID:" | cut -d' ' -f3)
+    
+    echo "Daemon $i - Team ID: $TEAM_ID"
+    TEAM_IDS[$i]=$TEAM_ID
+    
+    # Get device info
+    DEVICE_OUTPUT=$(aranya --uds-path /tmp/aranya$i/run/uds.sock -v device-info $TEAM_ID)
+    DEVICE_ID=$(echo "$DEVICE_OUTPUT" | grep "Device ID:" | cut -d' ' -f3)
+    IDENTITY_KEY=$(echo "$DEVICE_OUTPUT" | grep "Identity:" | sed 's/.*Identity: *//')
+    SIGNING_KEY=$(echo "$DEVICE_OUTPUT" | grep "Signing:" | sed 's/.*Signing: *//')
+    ENCODING_KEY=$(echo "$DEVICE_OUTPUT" | grep "Encoding:" | sed 's/.*Encoding: *//')
+    
+    echo "Daemon $i - Device ID: $DEVICE_ID"
+    DEVICE_IDS[$i]=$DEVICE_ID
+    IDENTITY_KEYS[$i]=$IDENTITY_KEY
+    SIGNING_KEYS[$i]=$SIGNING_KEY
+    ENCODING_KEYS[$i]=$ENCODING_KEY
+    
+    echo ""
+done
+
+echo "=== Adding devices 2-5 to team 1 ==="
+
+# Use team 1 as the main team
+MAIN_TEAM_ID=${TEAM_IDS[1]}
+echo "Main team ID (from daemon 1): $MAIN_TEAM_ID"
+
+# Add device 2 and assign admin role
+echo "Adding device 2 to team 1..."
+aranya --uds-path /tmp/aranya1/run/uds.sock \
+    add-device $MAIN_TEAM_ID ${IDENTITY_KEYS[2]} ${SIGNING_KEYS[2]} ${ENCODING_KEYS[2]}
```

Comment:
Using raw indexes is not very readable. You could assign a variable for each index to make the code more readable:
```suggestion
    add-device $MAIN_TEAM_ID ${IDENTITY_KEYS[${OPERATOR}]} ${SIGNING_KEYS[${OPERATOR}]} ${ENCODING_KEYS[${OPERATOR}]}
```